parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -428,4 +428,28 @@ write_files:
   encoding: gzip
   owner: root
   content: !!binary |
-    {{GetVariableProperty "cloudInitData" "customSearchDomainsScript"}}
\ No newline at end of file
+    {{GetVariableProperty "cloudInitData" "customSearchDomainsScript"}}
+
+{{if ShouldEnableLocalDNS}}
```

Comment:
@Devinwong How do we test custom data?

Reply:
Just checked the codes, it seems we don't have unit test for custom data now. If needed, it can be added in baker_test.go